webserver/decorators.py

```diff
@@ -15,6 +15,14 @@ def decorated(*args, **kwargs):
             raise Unauthorized
     return decorated
 
+def api_login_required(f):
```

Comment:
We actually already have an `auth_required` decorator above which is used for API endpoints. These two methods are exactly the same. We can reuse that one instead of making an new one